plural_rules.go

```diff
@@ -225,7 +225,6 @@ var singularToPluralSuffixList = []singularToPluralSuffix{
 	{"ode", "odes"},
 	{"ofe", "oves"},
 	{"pfe", "pves"},
-	{"pse", "psis"},
```

Comment:
cannot find an example of this type of transformation while "pse" to "pses" is common.